### DISCUSSION TITLE
[REQUEST] Linux Game Server Management (LGSM)

### DISCUSSION THREAD

**Discussion by drshockz** [tteck/Proxmox#2999] [Ideas]
Hey, I'm asking to request a LGSM container script that could setup any game of choice. LGSM is an amazing tool and a script to install the container and setup would be amazing. https://linuxgsm.com/ https://github.com/jesinmat/linux-gameservers

Thanks, and awesome site!

**Comment by tteck** [comment]
Use the [TurnKey LXC script](https://helper-scripts.com/scripts?id=4fgj5blht3hrbux) and select Game Server, it is [LinuxGSM](https://linuxgsm.com/)
